packages/@snowpack/plugin-build-script/plugin.ts

```diff
@@ -47,4 +59,6 @@ export function buildScriptPlugin(
       }
     },
   };
-}
+};
+
+export default buildScriptPlugin;
```

Comment:
This was moved to be a TypeScript file because Snowpack still imports this, and it needs the types. So this is our first plugin that requires building before deploying (otherwise, we’d have to manually manage types).